### PR TITLE
added splash button

### PR DESCRIPTION
**Pull request by ub2906**
Issue:

### CODE REVIEW

**Comment by ub2906** [comment]
I have created a splash button which when clicked on, creates a moving linear colored gradient in the button

**Review by shahednasser — CHANGES_REQUESTED** [review]
I like your idea, but can we make the animation continuous instead of on `:active`? I think it will look better that way.

**Comment by ub2906** [comment]
Hi! Thanks for the quick review. I made the animation appear on hover instead of active is that ok? Or should the black buttons be replaced as well?

**Comment by ub2906** [comment]
I just added a new commit as per my understanding. Hope it is better now :)

File: src/components/animated/_splash.less [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.splash-btn {
+  width: 220px;
+  height: 50px;
+  border: none;
+  outline: none;
+  color: #fff;
+  cursor: pointer;
+  position: relative;
+  z-index: 0;
+  border-radius: 10px;
```
Comment (shahednasser) [inline src/components/animated/_splash.less:10]: Can we make the border radius only for the buttons that don't have class `rounded-btn`? I think this is the only change we need to make. The rest looks good

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: src/components/animated/_splash.less (hunk 1 above)
Comment (ub2906) [reply src/components/animated/_splash.less]: Done!

**Comment by ub2906** [comment]
Done!

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @ub2906 for code and idea

**Pull request merged**